src/holodeck/environments.py

```diff
@@ -425,6 +425,60 @@ def add_agent(self, agent_def, is_main_agent=False):
         if is_main_agent:
             self._agent = self.agents[agent_def.name]
 
+    def spawn_prop(self, prop_type, location=None, rotation=None, scale=1, 
+                    sim_physics=False, material="", name=""):
+        """Spawns a basic prop object in the world like a box or sphere.
+
+        Args:
+            prop_type (:obj:`string`):
+                The type of prop to spawn. Can be ``box``, ``sphere``, ``cylinder``, or ``cone``.
+
+            location (:obj:`list` of :obj:`float`):
+                The ``[x, y, z]`` location of the prop.
+
+            rotation (:obj:`list` of :obj:`float`):
+                The ``[roll, pitch, yaw]`` rotation of the prop.
+
+            scale (:obj:`list` of :obj:`float`) or (:obj:`float`):
+                The ``[x, y, z]`` scalars to the prop size, where the default size is 1 meter.
+                If given a single float value, then every dimension will be scaled to that value.
+
+            sim_physics (:obj:`boolean`):
+                Whether the object is mobile and is affected by gravity and collisions.
```

Comment:
Wait I thought if this was off it would still collide, but not fall (gravity)

Reply:
I mean it is affected by collisions (knocked around when hit) as opposed to static, but I will clarify that